plugins/org.jkiss.dbeaver.net.ssh.sshj/src/org/jkiss/dbeaver/model/net/ssh/SSHJSessionController.java

```diff
@@ -64,6 +66,7 @@ protected SSHClient createNewSession(
         client.setConnectTimeout(connectTimeout);
         client.getConnection().getKeepAlive().setKeepAliveInterval(keepAliveInterval);
         client.getTransport().getConfig().setLoggerFactory(new FilterLoggerFactory());
+        client.getTransport().setTimeoutMs(10000);
```

Comment:
```suggestion
        client.getTransport().setTimeoutMs(SSHConstants.DEFAULT_CONNECT_TIMEOUT);
```

Reply:
Oops. It was a line added during debugging, we don't need it (setConnectTimeout does all the job)